source/basic.tex

```diff
@@ -300,38 +300,38 @@
 an unevaluated operand\iref{expr.prop},
 a subexpression thereof, or
 a conversion in an initialization or conversion sequence in such a context.
-The set of \defn{potential results} of an expression \tcode{e} is
+The set of \defn{potential results} of an expression $E$ is
 defined as follows:
 \begin{itemize}
-\item If \tcode{e} is an
+\item If $E$ is an
 \grammarterm{id-expression}\iref{expr.prim.id}, the set
-contains only \tcode{e}.
-\item If \tcode{e} is a subscripting operation\iref{expr.sub} with
+contains only $E$.
+\item If $E$ is a subscripting operation\iref{expr.sub} with
 an array operand, the set contains the potential results of that operand.
-\item If \tcode{e} is a class member access
+\item If $E$ is a class member access
 expression\iref{expr.ref} of the form
-\tcode{e1 . \opt{template} e2}
+\tcode{$E_1$ . \opt{template} $E_2$}
 naming a non-static data member,
-the set contains the potential results of \tcode{e1}.
-\item If \tcode{e} is a class member access expression
+the set contains the potential results of $E_1$.
+\item If $E$ is a class member access expression
 naming a static data member,
 the set contains the \grammarterm{id-expression} designating the data member.
-\item If \tcode{e} is a pointer-to-member
+\item If $E$ is a pointer-to-member
 expression\iref{expr.mptr.oper} of the form
-\tcode{e1 .* e2},
-the set contains the potential results of \tcode{e1}.
-\item If \tcode{e} has the form \tcode{(e1)}, the set contains the
-potential results of \tcode{e1}.
-\item If \tcode{e} is a glvalue conditional
+\tcode{$E_1$ .* $E_2$},
+the set contains the potential results of $E_1$.
+\item If $E$ has the form ($E_1$), the set contains the
```

Comment:
We lost the `\tcode` for the parens here.

Reply:
Fixed.